### PR TITLE
Add prod deploy

### PR DESCRIPTION
**Pull request by mathieud**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -30,10 +38,34 @@ jobs:
       - checkout
       - node/with-cache:
           steps:
-            - run: ./scripts/init-env.sh STAGING_
+            - run:
+                command: |
+                  if [ $CIRCLE_BRANCH = "staging" ]; then
```
Comment (maxb94) [inline .circleci/config.yml:43]: if the environment variables are preserved between commands, you should set PREFIX, ENV, and TARGET in one conditional

Comment (mathieud) [reply .circleci/config.yml:43]: I don't think they are.

Comment (maxb94) [reply .circleci/config.yml:43]: why not making it a single command then

Comment (mathieud) [reply .circleci/config.yml:45]: Confirmed (https://circleci.com/gh/sparkletown/sparkle/938?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link).

The environment setup is the same for the functions and hosting, that's why it's separated from the the actual deploy part. I can group them.

**Comment by maxb94** [comment]
ok

**Pull request merged**